kubeadm/v1.16.0/KubeCluster.ps1

```diff
@@ -0,0 +1,364 @@
+<#
```

Comment:
Will it make sense to refactor the structure here to share code between the different k8s versions? Looks like a lot of the 1.16 additions share common code with 1.15.

Reply:
> Will it make sense to refactor the structure here to share code between the different k8s versions? Looks like a lot of the 1.16 additions share common code with 1.15.

I was thinking about that when I was making the checkin..  The diff between 1.15 and 1.16 at this point is really just settings in the config files.  There's defaults in the scripts that change, but that's overridden by the config files.  At some point in the future there might be a wider diff in flags/etc needed though...

My initial thought was to just remove the defaults from the scripts entirely and have them be version agnostic (with additional errors if values weren't passed in in the config files), but wasn't sure if that'd make things too complicated down the road if there needed to be actual script differences between versions.

> I was not sure which changes in this PR are specifically needed for 1903.

Ahh sorry, should have been more clear in the PR notes there.  The 1903-specific support is lines 451/452 of KubeClusterHelper.psm1.  In 1903+ apparently the NICs used by HNS switch to a new LWVNIC model that doesn't show up in Win32_NetworkAdapterConfiguration, so need to be discovered via this registry node.  I verified that the new method still works for older windows versions (so not breaking backwards compatibility).

Reply:
I'll reorganize so the scripts are in the root level of the repo and just the configs are in the version-specific directories (since they're all that's actually version-specific at this point).